compass/ocean/tests/overflow/nonhydro/initial_state.py

```diff
@@ -0,0 +1,128 @@
+import time
```

Comment:
If this file is the same as `nonhydro_vs_hydro/initial_state`, let's just retain one of those files and import it for use in the other test case. You might even move it up one directory and rename my `initial_state` step to `initial_state_from_init_mode` or something like that.

Reply:
I moved `initial_state` up one directory and renamed yours `initial_state_from_init_mode` as you suggested.